modules/standard/Time.chpl

```diff
@@ -589,9 +597,9 @@ module Time {
   @chpldoc.nodoc
   proc date._chpldoc_workaround() { }
 
-  /* Writes this `date` in ISO 8601 format: YYYY-MM-DD */
+  /* Writes this `date` in format equivalent to cast to string */
```

Comment:
Ah I didn't think about how the docs would look. I am ok leaving it.

Reply:
It would be nice to have it there, but I think this is enough of a pointer for users to understand and means we don't duplicate it across our docs. What would be really nice is a link that takes you straight to the cast, which could be ``:operator:`date.:(type t = string) <date.:>` ``. But since this just links to any date cast operator, it might be too fragile.

Reply:
It would be nice to have it there, but I think this is enough of a pointer for users to understand and means we don't duplicate it across our docs. What would be really nice is a link that takes you straight to the cast, which could be `` :operator:`date.:(type t = string) <date.:>`  ``. But since this just links to any date cast operator, it might be too fragile.

Reply:
I was on the fence about whether or not to just put the format representation string in all the comments, as it's probably the clearest route and is unlikely to change. I'm waffling on it a bit, and kind of leaning towards just duplicating it.

I did try the link to the cast operator (had to use `:proc:`) but yeah it's just to any `date` cast. FWIW in this case this is the only `date` cast, but I do still think it's too brittle.

Reply:
Will do